include/libtorrent/disk_buffer_holder.hpp

```diff
@@ -69,7 +67,7 @@ namespace libtorrent
 	// 
 	// ``get()`` returns the pointer without transferring ownership. If
 	// this buffer has been released, ``get()`` will return nullptr.
-	struct TORRENT_EXPORT disk_buffer_holder
+	struct TORRENT_EXTRA_EXPORT disk_buffer_holder
```

Comment:
disk_buffer_holder is actually passed into one of the functions in peer_plugin, which means it needs to be exported.